package.json

```diff
@@ -46,25 +46,25 @@
     "stylelint-scss": "5.0.1",
     "stylelint-use-nesting": "4.1.0",
     "tslib": "^2.2.0",
-    "zone.js": "~0.11.4"
+    "zone.js": "~0.13.3"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^15.2.10",
-    "@angular-eslint/builder": "15.2.1",
-    "@angular-eslint/eslint-plugin": "15.2.1",
-    "@angular-eslint/eslint-plugin-template": "15.2.1",
-    "@angular-eslint/schematics": "15.2.1",
-    "@angular-eslint/template-parser": "15.2.1",
-    "@angular-material-components/datetime-picker": "^9.0.0",
-    "@angular/cli": "^15.2.10",
-    "@angular/compiler-cli": "^15.2.10",
-    "@angular/material-date-fns-adapter": "15.2.9",
+    "@angular-devkit/build-angular": "^16.2.12",
+    "@angular-eslint/builder": "16.3.1",
+    "@angular-eslint/eslint-plugin": "16.3.1",
+    "@angular-eslint/eslint-plugin-template": "16.3.1",
+    "@angular-eslint/schematics": "16.3.1",
+    "@angular-eslint/template-parser": "16.3.1",
+    "@angular-material-components/datetime-picker": "16.0.1",
+    "@angular/cli": "^16.2.12",
+    "@angular/compiler-cli": "^16.2.12",
+    "@angular/material-date-fns-adapter": "16.2.14",
     "@ckeditor/ckeditor5-angular": "^5.2.0",
     "@ckeditor/ckeditor5-build-classic": "^37.0.0",
     "@enigmatry/scss-foundation": "15.0.0-preview.5",
```

Comment:
Not sure why are we installing @enigmatry/scss-foundation and @enigmatry/stylelint-config as devDependencies